examples/resources/datadog_user_role/import.sh

```diff
@@ -0,0 +1,3 @@
+# This resource is imported using user_id and role_id seperated by `:`.
+
+terraform import datadog_user_role.foo "${user_id}:${role_id}"
```

Comment:
```suggestion
terraform import datadog_user_role.foo "${role_id}:${user_id}"
```
Looks like this needs to be reversed: